common/error.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
```

Comment:
if new files are added please make sure to update the year to 2024

Reply:
Done.